src/module/cache-container/CacheContainer.ts

```diff
@@ -43,7 +43,7 @@ module.config(($stateProvider: ng.ui.IStateProvider) => {
     url: "/caches",
     templateUrl: "module/cache-container/caches/view/caches.html",
     controller: CachesCtrl,
-    controllerAs: "cachesCtrl",
+    controllerAs: "$ctrl",
```

Comment:
The idea was to suggest using the default $ctrl alias throughout the app. I didn't want to touch all instances at one.
Since I presume there will be shift to ng4 at one time, and that currently only component based architecture will be used (or at least for each state it's corresponding template and ctrl) unique ctrl names are no longer necessary and may add to confusion as what a ctrl object is.